packages/create-project/commands/init/questions.js

```diff
@@ -6,18 +6,15 @@ module.exports = () => [
     choices: [
       {
         name: 'A web or Node.js application',
-        value: 'application',
-        checked: true
```

Comment:
Would omitting the `checked` property default to having the first option checked? Not having at least one checked property would maybe require the user to press the keyboard down arrow before seeing one of the options selected.

Reply:
Yes, this causes the first option to be selected. I think checked only works for checkable items, which these weren't.